keycloak/keycloak-operator
# Create OLM structure for 15.1.0

#415

## Related GH Issue
Closes #415 

## Additional Information
To generate structure to add to the PR to publish in OperatorHub.io

## Verification Steps
1. Build bundle and test catalog
2. deploy it in OCP
3. search for keycloak operator in operators , check it comes from test catalog
4. install it
5. check keycloak version is 15.1.0

## Checklist:
